generators/cucumber/templates/cuke_sniffer.rake

```diff
@@ -0,0 +1,21 @@
+require 'cuke_sniffer'
+
+def path_to_features
+  @_path_to_features ||= File.expand_path(File.join(__dir__, '..', 'features'))
+end
+
+def cuke_sniffer
+  @_cuke_sniffer ||= CukeSniffer::CLI.new(
+    features_location: path_to_features,
+    step_definitions_location: File.join(path_to_features, 'step_definitions'),
+    hooks_location: File.join(path_to_features, 'support'),
```

Comment:
I'm not sure but, it looks like that `:hooks_location` property should contain path to hooks.rb
`File.join(path_to_features, 'support', 'hooks.rb')`, isn't it?